lib/src/config/config_file/mod.rs

```diff
@@ -438,7 +438,7 @@ pub struct ValidatorSettings {
 
 #[derive(Clone, Debug, Deserialize, Default)]
 #[serde(deny_unknown_fields)]
-pub struct ZKPSettings {
+pub struct ZKProverSettings {
     #[serde(default)]
     pub prover_active: bool,
```

Comment:
I'd remove this bool and assume that if `zkp` is `None`, then the prover is not active.